helix-term/test.txt

```diff
@@ -0,0 +1,2 @@
+./Cargo.toml
+./
```

Comment:
I think you accidentally committed the testing file.

Reply:
Yes, will remove it!